cla_backend/libs/eligibility_calculator/cfe_civil/conversions.py

```diff
@@ -2,8 +2,11 @@
 
 
 def pence_to_pounds(pence):
-    decimal_value = (Decimal(pence) / 100).quantize(Decimal(".01"))
-    return float(decimal_value)
+    if pence is None:
+        return 0
```

Comment:
I'm not a fan of setting a blanket default value of 0 for all monetary figures. Feels more in control if we do it on a per-field basis. When you call this function you could add the default with `or 0`. e.g. in employment.py: `"tax": -pence_to_pounds(deductions.income_tax or 0),`

Reply:
@starswan